src/__tests__/utils/autoeqRegexTest.ts

```diff
@@ -0,0 +1,64 @@
+import fs from 'fs';
+import { PREAMP_REGEX, FILTER_REGEX } from '../../common/constants';
+
+const headphones = fs.readdirSync('autoeq');
+
+headphones.forEach((headphone) => {
+  const headphonePath = `autoeq/${headphone}`;
+  const eqFiles = fs.readdirSync(headphonePath);
+  eqFiles.forEach((eqFile) => {
+    const eqFilePath = `${headphonePath}/${eqFile}`;
+    const eqParams = fs.readFileSync(eqFilePath, { encoding: 'utf8' });
+    const eqLines = eqParams.split('\n');
+    eqLines.pop();
+
+    // console.log(eqParams);
+
+    const preampMatch = eqLines[0].match(PREAMP_REGEX);
+    if (preampMatch === null || preampMatch.length !== 2) {
+      throw new Error(
+        `Preamp regex match error for AutoEQ file: ${eqFilePath}`
+      );
+    } else {
+      let preamp = 0.0;
+      try {
+        preamp = parseFloat(preampMatch[1]);
```

Comment:
If we don't plan on using this value in the long run, could we change this so it calls `parseFloat(preampMatch[1])` without saving the returned value? We could do something similar to the later `try/catch` blocks too